examples/keras_io/vision/image_classifier.py

```diff
@@ -0,0 +1,679 @@
+# -*- coding: utf-8 -*-
+"""Classification in Keras-CV
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1qvQbaRuuMKJzp7cYgEQ7lY6kvBzglIqj
+
+# Classification with KerasCV
+
+**Author:** [lukewood](https://lukewood.xyz)<br>
```

Comment:
This is using markdown formatting -- don't. See other files for examples

Reply:
Done in #608.